src/app/shared/utils/common.ts

```diff
@@ -93,3 +93,8 @@ export function verifyJSON(data: string): boolean {
     return false;
   }
 }
+
+// eslint-disable-next-line @typescript-eslint/no-magic-numbers
```

Comment:
Remove this and handle this properly.

Reply:
after storing the default value in `maxUsageDefault` const, IDE is not showing the actual number, but the variable's name,

now: 
<img width="412" alt="image" src="https://user-images.githubusercontent.com/56511988/182113013-18b84335-2047-41ba-8f60-ffc768fe80ea.png">

before:
<img width="412" alt="image" src="https://user-images.githubusercontent.com/56511988/182113116-4cbc4e3d-1380-4a39-ba92-52bc283767cf.png">